src/ucs/datastruct/lru.h

```diff
@@ -33,25 +34,28 @@ KHASH_INIT(ucs_lru_hash, uint64_t, ucs_lru_element_t, 1, kh_int64_hash_func,
 /* Hash table type for LRU cache */
 typedef khash_t(ucs_lru_hash) ucs_lru_hash_t;
 
+/* User callback to get a single item from the cache.
+ * Used with @ref ucs_lru_get_all
+ * @param [in]  item    Item returned from the cache.
+ * @param [in]  opaque  User data. This object can be passed to
+ *                      @ref ucs_lru_get_all.
+ * @return UCS_OK if successful, or an error code as defined by
+ * @ref ucs_status_t otherwise.
+ * */
+typedef ucs_status_t (*ucs_lru_get_cb_t)(void *item, void *opaque);

```

Comment:
Can make `item` const? As it is a getter..

Reply:
removed